app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -53,13 +60,13 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:cursorVisible="true"
+      app:layout_goneMarginTop="28dp"
       android:inputType="@{viewModel.showPassword ? InputType.TYPE_CLASS_NUMBER : (InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_PASSWORD)}"
```

Comment:
itemRadius is `2dp` not `4dp`
lineColor is also different in mocks, `#00000029` this code is from mocks, you will need to convert this for android.

Reply:
I tried this color. It looks too light.

Reply:
Color code should be `#4A000000`

Reply:
done.